src/posix/platform/logging.c

```diff
@@ -46,7 +46,7 @@ void platformLoggingInit(const char *aName)
 #if (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED) || \
     (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_NCP_SPINEL)
 
-    openlog(aName, LOG_PID, LOG_USER);
+    openlog(aName, LOG_PID, LOG_DAEMON);
```

Comment:
This is fine. But curious if this is needed/related to other changes?

Reply:
I think mostly posix-app will be used as/in a daemon, so I changed this. I can update the description or title to indicate the change.

Reply:
It's ok. I was just curious about it.